woocommerce/woocommerce
# [Accessibility] Add focus indicator to image on product card

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #51839 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

**Testing shop page**
1. Go to the Shop page and tab-navigate through the products.
2. Verify the product images are focused with an outline.

**HEADS-UP 1**: Some themes have their own markup for the product card. Therefore, it's possible that the outline does not wrap only the image.

**Example**
Storefront theme:
![image](https://github.com/user-attachments/assets/3009e153-f7d7-42dd-9dea-b297e8e03229)

**HEADS-UP 2**: THe style of the focus outline style may vary as some themes have their own stylesheet.


**Testing the "You may be interested in..." section**
1. Make sure your Cart page uses the block variation.
4. Go to some product page on the admin.
5. Go to "Linked Products" and add some products to the "Cross-sells" box.
![Screenshot 2024-10-14 at 11 18 19](https://github.com/user-attachments/assets/4a0a2865-f552-49cc-8906-46fb095305db)
7. On the frontend, add the product to the cart.
8. Go to the Cart page and scroll down to the "You may be interested in..." section.
9. Focus on the product image and verify it has an outline.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
